framework/src/problems/FEProblemBase.C

```diff
@@ -1192,10 +1192,20 @@ FEProblemBase::initialSetup()
 
   // Perform Reporter get/declare check
   _reporter_data.check();
+  checkDuplicatePostprocessorVariableNames();
 
   setCurrentExecuteOnFlag(EXEC_NONE);
 }
 
+void
+FEProblemBase::checkDuplicatePostprocessorVariableNames()
+{
+  for (auto pp : _reporter_data.getPostprocessorNames())
```

Comment:
```suggestion
  for (const auto & pp : _reporter_data.getPostprocessorNames())
```
might as well not copy the strings